### ISSUE TITLE
Custom Header does not work with iobroker

### ISSUE THREAD

**Issue by sHamm3r** [ioBroker/ioBroker.lovelace#104]
Hello,

it would be really cool if you could make the Custom Header work. 
Enabeling this functionality would give a boost to this already very cool adapter.

I always end up with some strange js errors whenever I try to integrate this custom card.

Br Martin

**Comment by Garfonso** [comment]
could you please add more info:

- copy / screenshot of js errors
- version of lovelace adapter and card (if possible to tell)
- link to card
- what does not work (except from errors)

**Comment by sHamm3r** [comment]
Thats fast.

![grafik](https://user-images.githubusercontent.com/70481568/92304045-d19d0a80-ef7a-11ea-90db-9483873aed8b.png)

Lovelace 1.2.6
Custom-Header 1.7.6 [https://github.com/maykar/custom-header](https://github.com/maykar/custom-header) 

I basically try to change some attributes on the tabs like font-size and would like and would like to use some of the other features of the custom header. [https://maykar.github.io/custom-header/#intro](url)

I would like to show lovelace on an echo show 8, but the icons and text are to small. I can modify those styles on card level, but I'm not able to overwrite the styles on the header (tab) section.

Basically nothing works as the custom card already has issues during loading.
Today I'm already using the

card-mod.js
card-tools.js
layout-card.js
multiple-entity-row.js

with very good results, but the tabs are by far to small
![grafik](https://user-images.githubusercontent.com/70481568/92304219-445ab580-ef7c-11ea-8e74-5133b8c86777.png)

**Comment by Garfonso** [comment]
Thanks. I will look into it soonish.

I tried a bit, but it does not seem to be possible to change font-size using templates. But maybe there is a way? It might be possible to use custom CSS.

**Comment by sHamm3r** [comment]
Hi. CSS already works for cards using card-mod.js

views:
  - path: default_view
    style: |
      html {
        font-size: 30px;
      }
    title: Home
    panel: true
    badges: []
    cards:
      - type: 'custom:layout-card'
        layout: grid
        gridrows: 770px
        gridcols: 425px 425px auto
        cards:
          - type: entities
            gridrow: 1 / 4
            gridcol: 1 / 2
            style: |
              ha-card {
                height: 100%;
                font-size: 20px;
                --mdc-icon-size: 40px
              }
              .card-header {
                font-size: 30px;
              }
            title: Szenen

I run a homeassistant installation as reference to test locelace features and was able to change font on header level as well.

custom_header:
  header_text: custom header
  all_tabs_css: 'font-size: 25px;'

![grafik](https://user-images.githubusercontent.com/70481568/92322569-5434d100-f032-11ea-8258-0db5f498ed7c.png)

The lovelace adapter is my 1st choice, when it comes to ioBroker visualisation, because I like the clean look.

I'm not so familar with themes yet. Would it be possible to change only the font-size by using a theme.

I also like the other capabilities of the custom-header besides changing the style.

**Comment by sHamm3r** [comment]
It looks like custom-header.js is closely related to the used lovelace version. Which lovelace version is used in the ioBroker Adapter as base?

**Comment by sHamm3r** [comment]
I installed version 1.2.6 of the custom-header.js which seems to be corresponding to the ioBroker lovelace version.
Now I'm ending up with this error

![grafik](https://user-images.githubusercontent.com/70481568/92323017-88f65780-f035-11ea-86cb-0fa1dcb3a630.png)

https://github.com/maykar/custom-header/releases/tag/1.2.6

The error makes sense, because ioBroker does not have a sidebar. Is it possible to fork the custom-header.js to not use the sidebar. Js is not my hometurf, so I will most likely not be able to do it.

**Comment by Garfonso** [comment]
Since 1.2.* we use Version 2020.06.20 of Lovelace. 

For a workaround using custom css something like this might work: 
https://community.home-assistant.io/t/how-to-lovelace-css/97125/8

I.e. create a CSS file that set's the font-size you want (maybe for body or try to find out how to select the elements in the header, I'm always stumbling there), then upload it similar to a custom card. 

But I'll try to look into this, when I find time.
